microsoft-edge/webdriver-chromium/index.md

```diff
@@ -64,7 +64,7 @@ To begin writing automated tests, make sure the Edge WebDriver version you insta
 <!-- ====================================================================== -->
 ## Choose a WebDriver testing framework
 
-After downloading Edge WebDriver, the last component you must download is a WebDriver testing framework.  Test authors use WebDriver testing frameworks to write end-to-end tests and automate browsers.  A WebDriver testing framework provides a language-specific interface that translates your code into commands that Edge WebDriver runs in Microsoft Edge.  WebDriver testing frameworks exist for all major platforms and languages, such as Python, Java, C#, Ruby, and JavaScript.
+After downloading Edge WebDriver, the last component you must download is a WebDriver testing framework.  Test authors use WebDriver testing frameworks to write end-to-end tests and automate browsers.  A WebDriver testing framework provides a language-specific interface that translates your code into commands that Edge WebDriver runs in Microsoft Edge.  WebDriver testing frameworks exist for all major platforms and languages, such as Python, Java, C#, Ruby, and JavaScript. Note that for Python users, Python 3 is required and Python 2.7 is not supported.
```

Comment:
Please move the note to the "Using Selenium 4" section. Python 3 is only required if you want to follow our Selenium 4 examples.

Technically, you can use a WebDriver client written in any language, including Python 2.7, and we don't place restrictions on what client you use and what language it's written in as long as it is W3C compliant.

Reply:
Moved notes to `Using Selenium 4` sesion.